### ISSUE TITLE
The util namespace

### ISSUE THREAD

**Issue by amelchio** [amirziai/flatten#66]
I am unable to use `flatten_json` in a project that has its own `util`:
```text
$ (virtualenv --python=python3 flatten && cd flatten && source bin/activate && pip install flatten-json six && touch util.py && python -c "import flatten_json")
[...]
Installing collected packages: flatten-json, six
Successfully installed flatten-json-0.1.7 six-1.15.0
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/am/flatten/lib/python3.7/site-packages/flatten_json.py", line 13, in <module>
    from util import check_if_numbers_are_consecutive
ImportError: cannot import name 'check_if_numbers_are_consecutive' from 'util' (/Users/am/flatten/util.py)
```

(remove `touch util.py` from the shell line and it works without error)

I think the name `util` is a bit too generic to be reserved by this package. Is it possible to change that, maybe by using a relative import (`from .util import check_if_numbers_are_consecutive`)?

**Comment by 59e5aaf4** [comment]
Yeah, it feels weird to have that small 'util.py' in /usr/local/lib/python3.8/dist-packages/ . Looks like the package doesn't live in its own folder ?

```
    from flatten_json import flatten_json
  File "/usr/local/lib/python3.8/dist-packages/flatten_json.py", line 13, in <module>
    from util import check_if_numbers_are_consecutive
ImportError: cannot import name 'check_if_numbers_are_consecutive' from 'util'
```

I solved the problem by removing that useless util.py file and hardcoding its contents (the single function check_if_numbers_are_consecutive) in flatten_json.py

**Comment by TheMatt2** [comment]
> I solved the problem by removing that useless util.py file and hardcoding its contents (the single function check_if_numbers_are_consecutive) in flatten_json.py

I had to do the same thing.

I can't think of a good reason why this shouldn't be changed to `flatten_json.util` at the very least.

**Comment by amirziai** [comment]
i've remove `util.py` and just released `0.1.13`. please upgrade and lmk if you still have issues with it.

**Comment by TheMatt2** [comment]
@amirziai I think its safe to close this bug report out.
util has been refactored away
